### DISCUSSION TITLE
Prisma getPayload

### DISCUSSION THREAD

**Discussion by julien-e** [blitz-js/blitz#2115] [General]
Hello,

I try to get typesof model from prisma like Prisma.ModelGetPayload but don't working.

Do you have one solution ? for don't rewrite all model types in zod object.

exemple
```
import { resolver, Ctx } from "blitz"
import db from 'db'
import { Prisma } from 'db'
import * as z from "zod"

const CreateArticle = z
  .object(Prisma.ArticleCreateInput)
  .nonstrict()
```

Thank you

**Comment by flybayer** [comment]
Currently it's recommended to explicitly add the model fields to your zod schema. This is a bit of extra work, but it's almost always the right choice in the end for security. Because otherwise it would be too easy to add a sensitive field in prisma schema, like `admin: boolean` and then anyone would be able to update that
